Penteract/Assets/Scripts/Player.cpp

```diff
@@ -171,7 +171,7 @@ bool Player::GetInputBool(InputActions action, bool useGamepad) {
 		if (useGamepad && Input::IsGamepadConnected(0)) {
 			return Input::GetControllerAxisValue(Input::SDL_CONTROLLER_AXIS_TRIGGERLEFT, 0) > PRESSED_TRIGGER_THRESHOLD;
 		} else {
-			return Input::GetMouseButtonDown(2) || Input::GetMouseButtonRepeat(2);
+			return Input::GetKeyCodeDown(Input::KEYCODE::KEY_Q) || Input::GetKeyCodeRepeat(Input::KEYCODE::KEY_SPACE);
```

Comment:
Shouldn't this say KEYCODE::KEY_SPACE in both cases?

Reply:
Yes

Reply:
yes i dont know why it is key_q (i put key_space in both)